config/disks.php

```diff
@@ -27,7 +27,26 @@
     'use_https' => env('S3_UPLOADER_USE_HTTPS', env('S3_USE_HTTPS', env('AWS_USE_HTTPS', true))),
 ];
 
+$azureConfig = [
+    'driver' => 'azure',
+    'account' => [
+        'key' => env('AZURE_ACCOUNT_KEY'),
+        'name' => env('AZURE_ACCOUNT_NAME'),
+    ],
+    'container' => env('AZURE_CONTAINER', 'public'),
+    'endpoint-suffix' => env('AZURE_ENDPOINT_SUFFIX', 'core.windows.net'),
+    'protocol' => env('AZURE_UPLOADER_USE_HTTPS', env('AZURE_USE_HTTPS', true)),
```

Comment:
`protocol` feels weird being a Boolean value. Can we use `use_https` for consistency with the S3 driver?

Reply:
Yes I agree.